### PR TITLE
update observing squad

### PR DESCRIPTION
**Pull request by miiu96**

### CODE REVIEW

File: mainnet/run-proxy.sh [hunk 1]
```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-docker run --network=elrond-squad --ip=${IP} --name proxy elrondnetwork/elrond-proxy:v1.1.3
+docker run --network=elrond-squad --ip=${IP} --name proxy elrondnetwork/${PROXY_TAG}
```
Comment (andreibancioiu) [inline mainnet/run-proxy.sh:2]: From the README, the user doesn't know that it should also set `PROXY_TAG`:

```
export IP=10.0.0.2
./mainnet/run-proxy.sh
```

Perhaps add it there as well?

Of course, the user can `source .env` instead (but then we have to instruct the user to read the `.env` file carefully).

File: mainnet/run-observer.sh [hunk 2]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
-docker run --mount type=bind,source=${OBSERVER_DIR}/db,destination=/go/elrond-go/cmd/node/db --mount type=bind,source=${OBSERVER_DIR}/logs,destination=/go/elrond-go/cmd/node/logs --mount type=bind,source=${OBSERVER_DIR}/config,destination=/config --publish ${P2P_PORT}:37373 --network=elrond-squad --ip=${IP} --name squad-${SHARD} elrondnetwork/elrond-node-obs:v1.1.10 \
+docker run --mount type=bind,source=${OBSERVER_DIR}/db,destination=/go/elrond-go/cmd/node/db --mount type=bind,source=${OBSERVER_DIR}/logs,destination=/go/elrond-go/cmd/node/logs --mount type=bind,source=${OBSERVER_DIR}/config,destination=/config --publish ${P2P_PORT}:37373 --network=elrond-squad --ip=${IP} --name squad-${SHARD} elrondnetwork/${NODE_TAG} \
```
Comment (andreibancioiu) [inline mainnet/run-observer.sh:2]: From the README, the user doesn't know that it should also set `NODE_TAG` (when using the basic approach, not the docker compose):

```
export SHARD=2
export DISPLAY_NAME="MyObservingSquad-2"
export OBSERVER_DIR=~/MyObservingSquad/node-2
export P2P_PORT=10002
export IP=10.0.0.4
./mainnet/run-observer.sh
```

Perhaps update the readme?

File: mainnet/docker-compose.yml [hunk 3]
```diff
@@ -1,8 +1,8 @@
-version: '3.8'
+version: '3'
```
Comment (miiu96) [inline mainnet/docker-compose.yml:1]: no working for me with 3.8 ? should revert ?

**Review by andreibancioiu — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -8,8 +8,8 @@ Skip this if you prefer to pull from [docker hub](https://hub.docker.com/u/elron
 
 ```bash
 cd mainnet
-docker image build . -t elrondnetwork/elrond-node-obs:v1.1.10 -f ./elrond-node-obs
-docker image build . -t elrondnetwork/elrond-proxy:v1.1.3 -f ./elrond-proxy
+docker image build . -t elrondnetwork/elrond-node-obs:v1.1.24 -f ./elrond-node-obs
```
[the remark this answers is not in the document]
Comment (miiu96) [reply README.md:11]: done.

File: mainnet/run-proxy.sh (hunk 1 above)
Comment (miiu96) [reply mainnet/run-proxy.sh:2]: reverted

File: mainnet/run-observer.sh (hunk 2 above)
Comment (miiu96) [reply mainnet/run-observer.sh:2]: reverted

File: mainnet/docker-compose.yml (hunk 3 above)
Comment (miiu96) [reply mainnet/docker-compose.yml:1]: @andreibancioiu

Comment (andreibancioiu) [reply mainnet/docker-compose.yml:1]: It's okay, do not revert.

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**